tests/test_core.py

```diff
@@ -61,20 +62,20 @@ def test_upgrade_raises_template_not_found(repo: Repository):
         battenberg.upgrade()
 
 
-def test_upgrade_fetches_remote_template(installed_repo: Repository, template_repo: Repository):
-    # installed_repo.remotes.create('origin', 'git@github.com:zillow/battenberg.git')
-    template_oid = installed_repo.references.get('refs/heads/template').target
-    installed_repo.branches.remote.create('origin/template', installed_repo[template_oid])
-    installed_repo.branches.local.delete('template')
+# def test_upgrade_fetches_remote_template(installed_repo: Repository, template_repo: Repository):
```

Comment:
Why are you commenting out this test?

Reply:
Purely a mistake on my part, this couldn't run locally, and I forgot to re-add it.

Reply:
Purely a mistake on my part, this couldn't run locally, and I forgot to re-add it.